src/PKSim.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -69,6 +69,9 @@ public override async Task<SnapshotProject> MapToSnapshot(ModelProject project)
             x.Description = SnapshotValueFor(project.Description);
          });
 
+         //first pass. We load all building blocks to ensure that there are no issues with the building blocks loading order
+         project.All<IPKSimBuildingBlock>().Each(_lazyLoadTask.Load);
```

Comment:
@Yuri05 This is the fix. It seems more natural to me than saving individual and expression profile in this order. We normally define expression profile first so having to save them backward looks like a hack to me